src/main/java/service/Mode.java

```diff
@@ -0,0 +1,33 @@
+package service;
+
+import message.ExecuteMessage;
+import view.SelectMenu;
+import domain.ModeType;
+import exception.NotAppropriateScope;
+import repository.FileRepository;
+import repository.MemoryRepository;
+import java.util.InputMismatchException;
+
+import static domain.ConsolePrint.getMenuNum;
+
+public class Mode {
+    Service service;
+    public Mode(ModeType mode) {
+        if(mode == ModeType.NORMAL_MODE) service = new Service(new FileRepository());
+        else if(mode == ModeType.TEST_MODE) service = new Service(new MemoryRepository());
+    }
+
+
+    public boolean run() {
```

Comment:
```suggestion

    public Mode(ModeType mode) {
        if(mode == ModeType.NORMAL_MODE) service = new Service(new FileRepository());
        else if(mode == ModeType.TEST_MODE) service = new Service(new MemoryRepository());
    }

    public boolean run() {
```
이렇게 하는게 더 깔끔해 보일거 같아요.